intro-to-R-tidyverse/01-intro_to_base_R.Rmd

```diff
@@ -407,54 +411,55 @@ We can additionally explore _overall properties_ of the data frame with two diff
 
 This provides summary statistics for each column:
 
-```{r iris-summary}
-summary(iris)
+```{r penguins-summary}
+summary(penguins)
 ```
 
 This provides a short view of the **str**ucture and contents of the data frame.
 
-```{r iris-str}
-str(iris)
+```{r penguins-str}
+str(penguins)
 ```
 
-You'll notice that the column `Species` is a _factor_: This is a special type of character variable that represents distinct categories known as "levels". 
-We have learned here that there are three levels in the `Species` column: setosa, versicolor, and virginica. 
+You'll notice that the column `species` is a _factor_: This is a special type of character variable that represents distinct categories known as "levels". 
+We have learned here that there are three levels in the `species` column: Adelie, Chinstrap, and Gentoo.
 We might want to explore individual columns of the data frame more in-depth. 
 We can examine individual columns using the dollar sign `$` to select one by name:
 
-```{r iris-subset}
-# Extract Sepal.Length as a vector
-iris$Sepal.Length
+```{r penguins-subset}
+# Extract bill_length_mm as a vector
+penguins$bill_length_mm
 
 # indexing operators can be used too
-iris$Sepal.Width[1:10]
+penguins$bill_depth_mm[1:10]
 ```
 
 We can perform our regular vector operations on columns directly.
 
-```{r iris-col-mean, live = TRUE}
-# calculate the mean of the Sepal.Length column
-mean(iris$Sepal.Length)
+```{r penguins-col-mean, live = TRUE}
+# calculate the mean of the bill_length_mm column
+mean(penguins$bill_length_mm, 
```

Comment:
I'm debating about leaving it like this and just gliding right through this argument OR adding some more steps to this and asking the participants whether they notice that their are `NA`s in this set.

Reply:
I guess main question is how much time we want to devote to this `NA` thing.

Reply:
I think it is fine as is. This is a place where live coding should work well. 

Note that the comment you have here will be stripped out, as only full line comments are kept. In this case, I think this is a good thing! It means that somebody looking at the rendered notebook will see the full explanation, but live participants won't see it before somethin goes 'wrong'.

Reply:
Ah good point. I wasn't thinking about the `live`-ness part.